### ISSUE TITLE
Failed `conda install spacy` seems to break conda installation

### ISSUE THREAD

**Issue by phdowling** [conda/conda#2464]
This may relate to #2325 :
This happened to me today while running a regular conda install (on Max OSX Yosemite): 

```
Philipps-Air: dowling$conda install spacy
[...]
thinc-5.0.6-py 100% |############################################################################################################################################################| Time: 0:00:00 427.70 kB/s
spacy-0.100.6- 100% |############################################################################################################################################################| Time: 0:00:03 307.28 kB/s
Extracting packages ...
[      COMPLETE      ]|###############################################################################################################################################################################| 100%
Unlinking packages ...
[      COMPLETE      ]|###############################################################################################################################################################################| 100%
Linking packages ...
An unexpected error has occurred, please consider sending the                                                                                                                                         |  18%
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/Users/dowling/anaconda/bin/conda", line 5, in <module>
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 173, in main
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/cli/main.py", line 180, in args_func
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/cli/main_install.py", line 45, in execute
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/cli/install.py", line 423, in install
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/plan.py", line 538, in execute_actions
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/instructions.py", line 148, in execute_instructions
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/instructions.py", line 91, in LINK_CMD
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/instructions.py", line 87, in link
  File "/Users/dowling/anaconda/lib/python2.7/site-packages/conda/install.py", line 616, in link
  File "/Users/dowling/anaconda/lib/python2.7/os.py", line 150, in makedirs
    makedirs(head, mode)
  File "/Users/dowling/anaconda/lib/python2.7/os.py", line 157, in makedirs
    mkdir(name, mode)
OSError: [Errno 13] Permission denied: '/Users/dowling/anaconda/lib/python2.7/site-packages/murmurhash/headers'
```

I figured I could just run as root to fix this:

```
Philipps-Air: dowling$ sudo conda install spacy
sudo: conda: command not found
Philipps-Air: dowling$ sudo -H conda install spacy
sudo: conda: command not found
```

Then I figured maybe I could change the folder permissions to make this work:

```
Philipps-Air: dowling$ chmod -v 755 /Users/dowling/anaconda/
/Users/dowling/anaconda/
Philipps-Air: dowling$ conda install spacy
-bash: /Users/dowling/anaconda/bin/conda: No such file or directory
```

And now the `conda` file really appears to be gone. I don't even know what step of this would have caused that, but here we go. Any ideas on what happened here, or how to fix it?

**Comment by ijstokes** [comment]
You might be able to recover with the strategy documented here:

https://github.com/conda/conda/issues/1733#issuecomment-150641779

For a discussion of what might have happened, see the discussion here:

https://github.com/conda/conda/issues/2467

**Comment by kalefranz** [comment]
Permissions errors and multi-user/read-only installs have been fixed in later versions of conda 4.3.  Also fixed in 4.3. is conda uninstalling itself.
